frontend/package.json

```diff
@@ -34,5 +34,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "axios": "^0.21.1"
```

Comment:
axios should be a normal dependency and no dev dependency ;)

Reply:
Thanks!